website/addons/github/static/githubFangornConfig.js

```diff
@@ -176,7 +176,11 @@ var _githubItemButtons = {
                         },
                         icon: 'fa fa-external-link',
                         className: 'text-info'
-                    }, 'Open'),
+                    }, 'Open')
+                );
+            }
+            if (item.data.addonFullname) {
```

Comment:
All of the surrounding code here contains the condition `tb.options.placement !== 'fileview'` - should this one as well?

Also, see [my previous comment re: repeated conditions](https://github.com/CenterForOpenScience/osf.io/pull/3161/files#r32631325). It may be applicable here as well.